src/main/java/com/rapid7/client/dcerpc/objects/RPCReferentConformantArray.java

```diff
@@ -60,6 +62,7 @@ public void marshalEntity(PacketOutput out) throws IOException {
         if (array == null)
```

Comment:
Array is not allowed to be null. It would only be null if this class represented a pointer to a conformant array.

Reply:
Will fix.